components-core/src/main/java/com/adyen/checkout/components/core/internal/data/api/AnalyticsRepositoryData.kt

```diff
@@ -39,24 +40,53 @@ data class AnalyticsRepositoryData(
         componentParams = componentParams,
         source = AnalyticsSource.PaymentComponent(componentParams.isCreatedByDropIn, paymentMethod),
         paymentMethodType = paymentMethod.type,
+        sessionId = null,
+    )
+
+    constructor(
+        application: Application,
+        componentParams: ComponentParams,
+        paymentMethod: PaymentMethod,
+        sessionId: String,
```

Comment:
Looks like f you make this nullable and give it null as default value, the other 2 constructors are no longer needed

Reply:
Of course 🤦